src/websocket/client.ts

```diff
@@ -72,8 +72,8 @@ export class WsClient {
         socket.onmessage = async (encodedMsg: any) => {
           let data = encodedMsg.data
 
-          // If data is not a buffer, it will be a blob
-          if (!Buffer.isBuffer(data)) {
+          // If data is not a buffer (nodejs), it will be a blob (browser)
+          if (typeof Buffer === "undefined" || !Buffer.isBuffer(data)) {
```

Comment:
is there any need for this part anymore `!Buffer.isBuffer(data)`? was it just covering the browser case?

Reply:
Yeah that covers the case in which the web app injects the Buffer library. But since we are in the browser, the data will still be a blob.